src/main/java/seedu/linkedout/model/UserPrefs.java

```diff
@@ -51,11 +52,21 @@ public Path getAddressBookFilePath() {
         return addressBookFilePath;
     }
 
+    public Path getLinkedoutFilePath() {
+        // return linkedoutBookFilePath;
+        return addressBookFilePath;
```

Comment:
The method getLinkedoutFilePath() still uses the addressBookFilePath, maybe can update to linkedoutBookFilePath?
Not sure if the implementation will still work though. 
If you have time maybe can update this?

Reply:
Hi Ryan, I think its okay to comment it out for now, as linkedoutBookFilePath would point to a different file path and cause the program to crash. I'll update it when storage is done refactoring!